docs/api/en/objects/BatchedMesh.html

```diff
@@ -167,16 +167,27 @@ <h3>
 		<p>
 			[page:Integer instanceId]: The id of an instance to get the visibility state of.
 		</p>
-		<p>Get whether the given instance is marked as "visible" or not.</p>
+		<p>Get whether the given instance is marked as "visible" or not.</p>		
+
+		<h3>
+			[method:Object getGeometryRangeAt]( [param:Integer geometryId] )
+		</h3>
+		<p>
+			[page:Integer geometryId]: The id of the geometry to retrieve the geometry range.
```

Comment:
We should describe in a bit more detail what "geometry range" means. I don't think it would be clear to a less-advanced user that the returned value is a subrange of triangles of the attached BufferGeometry.